src/model_signing/_serialization/file_shard.py

```diff
@@ -87,6 +87,9 @@ def __init__(
         # None of the arguments used to build the hasher are used.
         hasher = sharded_hasher_factory(pathlib.Path(), 0, 1)
         self._shard_size = hasher.shard_size
+        self._serialization = manifest._ShardSerialization(
+            hasher.digest_name, self._shard_size, self._allow_symlinks
+        )
```

Comment:
similar optional around serialization desc/type

Reply:
Changed in the same way